### ISSUE TITLE
Could not Update  alacritty

### ISSUE THREAD

**Issue by blueray453** [alacritty/alacritty#5364]
### System

OS: Linux Mint 20.2
Version: alacritty 0.7.2
Linux/BSD: X11, Cinnamon / Mutter / metacity

### Logs

I was trying to update alacritty. I used `cargo install alacritty --verbose`. The error part of the output in given bellow.

```
error: aborting due to previous error

error: failed to compile `alacritty v0.8.0`, intermediate artifacts can be found at `/tmp/cargo-installwyYno7`

Caused by:
  could not compile `alacritty`

Caused by:
  process didn't exit successfully: `rustc --crate-name alacritty --edition=2018 /home/ismail/.cargo/registry/src/github.com-1ecc6299db9ec823/alacritty-0.8.0/src/main.rs --error-format=json --json=diagnostic-rendered-ansi --crate-type bin --emit=dep-info,link -C opt-level=3 -C embed-bitcode=no --cfg 'feature="default"' --cfg 'feature="png"' --cfg 'feature="wayland"' --cfg 'feature="wayland-client"' --cfg 'feature="x11"' --cfg 'feature="x11-dl"' -C metadata=695d436696b79806 -C extra-filename=-695d436696b79806 --out-dir /tmp/cargo-installwyYno7/release/deps -L dependency=/tmp/cargo-installwyYno7/release/deps --extern alacritty_config_derive=/tmp/cargo-installwyYno7/release/deps/libalacritty_config_derive-f51ecd60ff751ebf.so --extern alacritty_terminal=/tmp/cargo-installwyYno7/release/deps/libalacritty_terminal-0e836f5abd64d941.rlib --extern bitflags=/tmp/cargo-installwyYno7/release/deps/libbitflags-61ee16c26132f2e5.rlib --extern clap=/tmp/cargo-installwyYno7/release/deps/libclap-3ddd3cebfce1912b.rlib --extern copypasta=/tmp/cargo-installwyYno7/release/deps/libcopypasta-15e19fb4a07a42d4.rlib --extern crossfont=/tmp/cargo-installwyYno7/release/deps/libcrossfont-3126744595a9891c.rlib --extern dirs=/tmp/cargo-installwyYno7/release/deps/libdirs-de3fc9662af437b4.rlib --extern fnv=/tmp/cargo-installwyYno7/release/deps/libfnv-fb56affc7069367a.rlib --extern glutin=/tmp/cargo-installwyYno7/release/deps/libglutin-94d338a86e981edc.rlib --extern libc=/tmp/cargo-installwyYno7/release/deps/liblibc-3d50654642042bf2.rlib --extern log=/tmp/cargo-installwyYno7/release/deps/liblog-141ba0c74291a9ee.rlib --extern notify=/tmp/cargo-installwyYno7/release/deps/libnotify-242caca284915d02.rlib --extern parking_lot=/tmp/cargo-installwyYno7/release/deps/libparking_lot-d8a7474a609d67ab.rlib --extern png=/tmp/cargo-installwyYno7/release/deps/libpng-7733f1b6cf88cc2a.rlib --extern serde=/tmp/cargo-installwyYno7/release/deps/libserde-25483b49e4c8d51f.rlib --extern serde_json=/tmp/cargo-installwyYno7/release/deps/libserde_json-402abdbcab2ed666.rlib --extern serde_yaml=/tmp/cargo-installwyYno7/release/deps/libserde_yaml-09512854e2568f06.rlib --extern time=/tmp/cargo-installwyYno7/release/deps/libtime-1a21a481c0625ee8.rlib --extern unicode_width=/tmp/cargo-installwyYno7/release/deps/libunicode_width-b995ec368e728108.rlib --extern wayland_client=/tmp/cargo-installwyYno7/release/deps/libwayland_client-4ca335e76f66e357.rlib --extern x11_dl=/tmp/cargo-installwyYno7/release/deps/libx11_dl-6f14b599ca8e1248.rlib --extern xdg=/tmp/cargo-installwyYno7/release/deps/libxdg-f935a4b380969d41.rlib --cap-lints allow -L native=/usr/lib/x86_64-linux-gnu -L native=/usr/lib/x86_64-linux-gnu -L native=/usr/lib/x86_64-linux-gnu -L native=/usr/lib/x86_64-linux-gnu` (exit status: 1)
```

**Comment by chrisduerr** [comment]
Please check the installation instructions if you have issues with building Alacritty from source.

**Comment by blueray453** [comment]
I installed alacritty using `cargo install alacritty`. Is it considered building from source? 

Where can I find the documentation to update Alacritty?

**Comment by chrisduerr** [comment]
Yes, of course that is considered installing from source.

**Comment by blueray453** [comment]
Thank you very much. I was able to update. 

For those who are interested to know. I think the dependencies changed between 0.7.2 and 0.8.0

So, I installed the dependencies from https://github.com/alacritty/alacritty/blob/master/INSTALL.md#debianubuntu

Then by following https://stackoverflow.com/questions/68509475/what-is-the-command-to-update-all-my-rust-packages , ran:

```
cargo install cargo-update
cargo install-update -a
```
